narwhal/types/src/primary.rs

```diff
@@ -780,6 +840,8 @@ impl fmt::Display for BlockErrorKind {
 pub struct WorkerOurBatchMessage {
     pub digest: BatchDigest,
     pub worker_id: WorkerId,
+    /// The time when the batch was created
+    pub timestamp: TimestampMs,
```

Comment:
same comment about wanting some indication about "timestamp of what?" in the field name